src/extension.ts

```diff
@@ -62,6 +68,60 @@ const provideInitialConfigurations = (): string => {
   ].join('\n');
 };
 
+const checkPath = () => {
+  fs.stat(lastModified.filePath, async (err, stat) => {
+    if (lastModified.mtime < stat.mtime) {
+      await setup();
+      createTizenProject();
+    } else {
+      createTizenProject();
+    }
```

Comment:
The else case is unnecessary. You can call the createTizenProject after the if statement.